charts/harvester-csi-driver-lvm/values.yaml

```diff
@@ -14,6 +14,12 @@ provisionerImage:
   # Overrides the image tag whose default is the chart appVersion.
   tag: "v0.1.2"
 
+hookImage:
+  repository: rancher/kubectl
+  pullPolicy: IfNotPresent
```

Comment:
nit, the pullPolicy is not used in the template

Reply:
updated!